### ISSUE TITLE
@ember/string does not have a isHTMLSafe import

### ISSUE THREAD

**Issue by BorigTheDwarf** [Deveo/ember-emojione#48]
Running latest install of Ember (2.14.2) - can't serve after following the basic installation instructions due to the following error:

`SyntaxError: ember-emojione/helpers/inject-emoji.js: @ember/string does not have a isHTMLSafe import
> 1 | import { htmlSafe, isHTMLSafe } from '@ember/string';
    | ^
  2 | import Helper from '@ember/component/helper';
  3 | import { getProperties } from '@ember/object';
  4 | import config from 'ember-get-config';
`

**Comment by lolmaus** [comment]
Try throwing the polyfill in:

    ember install ember-string-ishtmlsafe-polyfill

**Comment by BorigTheDwarf** [comment]
I attempted this but it does not resolve the issue and adds the following warning:

`ember-string-ishtmlsafe-polyfill is not required for Ember 2.8.0 and later, please remove from your `package.json`.`

**Comment by lolmaus** [comment]
@Turbo87 Can you please look into this? It's definitely a result of your #47.

**Comment by BorigTheDwarf** [comment]
I downgraded to version 3.01 and am now getting this error:

```
SyntaxError: ember-emojione/helpers/inject-emoji.js: Unexpected token (51:6)
  49 | 
  50 |     return {
> 51 |       ...envOptions,
     |       ^
  52 |       ...overrideOptions,
  53 | 
  54 |       emojione: {
```

**Comment by Turbo87** [comment]
The issue is a locked ember-rfc176-data subdependency. It was fixed and released in a patch release but when people use lockfiles they obviously don't get any bug fix releases...

**Comment by lolmaus** [comment]
@Turbo87 So what's the solution? Run `yarn upgrade`?

**Comment by Turbo87** [comment]
@lolmaus we need to release another version of `ember-cli-babel` that enforces the rfc bugfix release and then also update the dependency here.

the bugfix was https://github.com/ember-cli/ember-rfc176-data/pull/23

/cc @rwjblue

**Comment by rwjblue** [comment]
I believe that this has been done, can you confirm?

**Comment by Turbo87** [comment]
@rwjblue https://github.com/ember-cli/babel-plugin-ember-modules-api-polyfill has not yet been released enforcing that subdep update

**Comment by Turbo87** [comment]
@lolmaus `ember-cli-babel@6.8.2` enforces the necessary bugfix now. if you update your dependency here and release a new patch release this issue should be fixed for you.

**Comment by lolmaus** [comment]
@Turbo87 @rwjblue Thank you. :bow:

Released `v3.0.3`.

@BorigTheDwarf, can you please try it out to see if it resolves your issue?
